pkg/server/datastore/sqlstore/migration.go

```diff
@@ -300,7 +300,7 @@ func migrateDB(db *gorm.DB, dbType string, disableMigration bool, log logrus.Fie
 	dbCodeVersion, err := getDBCodeVersion(*migration)
 	if err != nil {
 		log.WithError(err).Error("Error getting DB code version")
-		return sqlError.New("error getting DB code version: %v", err)
+		return newSQLError("error getting DB code version: %v", err)
```

Comment:
Would this be better in any way with `newWrappedSQLError(fmt.Errorf("error getting DB code version: %w", err))`?